epam-promo/components/inputs/docs/searchInput.doc.ts

```diff
@@ -1,11 +1,12 @@
 import { DocBuilder, isReadonlyDoc } from '@epam/uui-docs';
 import { TextInputProps } from '@epam/uui-components';
-import { TextInputMods, SearchInput } from '../TextInput';
+import { SearchInput } from '../TextInput';
 import {
     iEditable, sizeDoc, iHasPlaceholder, onClickDoc, isDisabledDoc, isInvalidDoc, iconDoc, iconOptionsDoc, dropdownTogglerDoc,
     DefaultContext, FormContext,
 } from '../../../docs';
 import { IEditableDebouncerOptions } from '@epam/uui-core';
+import { TextInputMods } from "@epam/uui";
```

Comment:
use here TextInputProps interface which union TextInputMods and core text input props

Reply:
done